opwen_email_server/config.py

```diff
@@ -25,3 +25,10 @@
         host=environ.get('LOKOLE_EMAIL_SERVER_QUEUES_NAMESPACE'))
 else:
     QUEUE_BROKER = environ.get('QUEUE_BROKER_URL', '')
+
+
+def str2bool(s):
+  return s.lower() in ("true")
+
+
+CREATE_MISSING = str2bool(environ.get('CREATE_QUEUES', 'TRUE'))
```

Comment:
This seems unused (and isn't the default behavior in Celery to create missing queues?) so can it be removed?

Reply:
Yes, default behavior is to create missing queues so it can indeed be removed.